### PR TITLE
Add example Dockerfile

### PR DESCRIPTION
**Pull request by davralin**
Attached is an example Dockerfile that will install news-updater from git, and execute it with the config-file from /data/config.ini.

Is it possible to add some automated builds for this under a more official name?

If so, I would be happy to add some documentation for running this as both a standalone container, or in kubernetes.

(for reference, I'm using this container in kubernetes now: `davralin/news-updater`.)

### CODE REVIEW

**Comment by BernhardPosselt** [comment]
Hi davralin, thanks for your pull request.

I don't think adding a Dockerfile is of much value. It's super simple and obvious how to do it but requires constant updates to stay up to date with the latest security updates. In a similar vain, we also don't provide distro packages. Your better off managing the container on your own.

**Comment by davralin** [comment]
> Hi davralin, thanks for your pull request.
> 
> I don't think adding a Dockerfile is of much value. It's super simple and obvious how to do it but requires constant updates to stay up to date with the latest security updates. In a similar vain, we also don't provide distro packages. Your better off managing the container on your own.

I see your point, but don't agree with your conclusion. :smile: 

The actual Dockerfile in itself does not require constant updates at all, and would just have to be updated at the same time as the Travis-pipeline-definition has to be upgraded.

As for how easy it is, that's in the eye of the beholder - and the rest of Nextcloud's ecosystem both publish and use containers - so I was surprised that this did not.
It might be easy, but >1 people doing it manually is still uneeded - and an inclusion of a Dockerfile can also just be viewed as documentation.

I also assume that Nextcloud have some sort of management system for maintaing updates to the containers they publish wrt. security updates - and if they don't, or you can't tap into that - I can understand your reluctance against publishing this somewhere.

To summarize, I can understand why you are hesitant to publish a ready-made image - but there are ways to address your (valid) concerns for this.

I still think the Dockerfile should be included however, if just to be viewed as an example.

**Comment by BernhardPosselt** [comment]
Why don't you simply provide the container on Dockerhub then?

**Comment by davralin** [comment]
I don't think it's fair to anyone that a container under nextcloud's "umbrella" is published under someone completely different.

But, since you don't want this merged I'll just close it. :-)
